pepper-apis/src/main/java/org/broadinstitute/ddp/json/FileUploadURLPayload.java

```diff
@@ -0,0 +1,63 @@
+package org.broadinstitute.ddp.json;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+import com.google.gson.annotations.SerializedName;
+
+public class FileUploadURLPayload {
+    @NotBlank
+    @SerializedName("fileName")
+    String fileName;
```

Comment:
We should probably constrain the length of the file name so we don't have problems inserting name or path afterwards. Something big but not too big.

Reply:
Set to 200